pkg/cosign/upload.go

```diff
@@ -34,10 +34,11 @@ import (
 )
 
 const (
-	ExperimentalEnv = "COSIGN_EXPERIMENTAL"
-	repoEnv         = "COSIGN_REPOSITORY"
-	ServerEnv       = "REKOR_SERVER"
-	rekorServer     = "https://api.rekor.dev"
+	ExperimentalEnv    = "COSIGN_EXPERIMENTAL"
+	repoEnv            = "COSIGN_REPOSITORY"
+	LegacyMediaTypeEnv = "COSIGN_LEGACY_MEDIA_TYPE"
```

Comment:
Want to just call it COSIGN_DOCKER_MEDIA_TYPE instead of legacy to make it more clear?

Reply:
done